datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -24,52 +24,53 @@ ddtrace==0.32.2; sys_platform == 'win32' and python_version < '3.0'
 ddtrace==0.53.2; sys_platform != 'win32' or python_version > '3.0'
 dnspython==1.16.0
 enum34==1.1.10; python_version < '3.0'
-foundationdb==6.3.24; python_version > '3.0'
+foundationdb==7.1.17; python_version > '3.0'
 futures==3.3.0; python_version < '3.0'
 gearman==2.0.2; sys_platform != 'win32' and python_version < '3.0'
 gssapi==1.6.1; python_version < '3.0'
 gssapi==1.7.3; python_version > '3.0'
-immutables==0.17; python_version > '3.0'
+immutables==0.18; python_version > '3.0'
 in-toto==1.0.1
 ipaddress==1.0.23; python_version < '3.0'
 jaydebeapi==1.2.3
 jellyfish==0.9.0; python_version > '3.0'
 jpype1==0.7.0; python_version < '3.0'
-jpype1==1.3.0; python_version > '3.0'
+jpype1==1.4.0; python_version > '3.0'
 kafka-python==2.0.2
 kazoo==2.8.0
 kubernetes==18.20.0; python_version < '3.0'
-kubernetes==22.6.0; python_version > '3.0'
+kubernetes==24.2.0; python_version > '3.0'
 ldap3==2.9.1
-lxml==4.8.0
+lxml==4.9.1
 lz4==2.2.1; python_version < '3.0'
-lz4==3.1.3; python_version > '3.0'
+lz4==4.0.2; python_version > '3.0'
 mmh3==2.5.1; python_version < '3.0'
 mmh3==3.0.0; python_version > '3.0'
-openstacksdk==0.39.0; python_version < '3.0'
-openstacksdk==0.61.0; python_version > '3.0'
-orjson==3.6.8; python_version > '3.0'
-paramiko==2.10.4
+openstacksdk==0.100.0; python_version > '3.0'
+openstacksdk==0.45.0; python_version < '3.0'
+orjson==3.7.11; python_version > '3.0'
+paramiko==2.11.0
 ply==3.11
 prometheus-client==0.12.0; python_version < '3.0'
 prometheus-client==0.14.1; python_version > '3.0'
-protobuf==3.17.3; python_version < '3.0'
-protobuf==3.20.1; python_version > '3.0'
-psutil==5.9.0
+protobuf==4.21.0; python_version < '3.0'
```

Comment:
Is this intentional? 4.21.0 was retracted from pypi. Also, since it's the same series as `>3.0` version, can we use the same protobuf version for both py2 and py3?

Reply:
Fixed